_templates/component/new/prompt.js

```diff
@@ -18,7 +18,13 @@ module.exports = {
         name: 'path',
         message: 'Where should this component be created?',
         choices: ['components', 'fields', 'providers']
-      }
+      },
+      {
+	        type: 'confirm',
+        name: 'isPayloadBlock',
+        message: 'Is this component going to be used as a Payload block?'
+
+	}
```

Comment:
Why isn't this just using teh above `path` logic?

`Block` might just need its own prompt